insights/specs/insights_archive.py

```diff
@@ -45,7 +45,7 @@ class InsightsArchiveSpecs(Specs):
     docker_info = simple_file("insights_commands/docker_info")
     docker_list_containers = simple_file("insights_commands/docker_ps_--all_--no-trunc")
     docker_list_images = simple_file("insights_commands/docker_images_--all_--no-trunc_--digests")
-    engine_db_query_vdsm_version = simple_file("insights_commands/engine-db-query_-s_SELECT_vs.vds_name_rpm_version_FROM_vds_dynamic_vd_vds_static_vs_WHERE_vd.vds_id_vs.vds_id_--json")
+    engine_db_query_vdsm_version = first_file(["insights_commands/engine-db-query_--statement_SELECT_vs.vds_name_rpm_version_FROM_vds_dynamic_vd_vds_static_vs_WHERE_vd.vds_id_vs.vds_id_--json", "insights_commands/engine-db-query_-s_SELECT_vs.vds_name_rpm_version_FROM_vds_dynamic_vd_vds_static_vs_WHERE_vd.vds_id_vs.vds_id_--json"])
```

Comment:
@psachin since the original spec was broken I don't think its necessary to use first_file here.  I think it is OK to ignore the files in archives before this was fixed.

Reply:
@bfahr Sure.